docker/docker-compose.yml

```diff
@@ -25,6 +25,7 @@ services:
       - "traefik.backend=gsuite-wrapper"
       - "traefik.port=8084"
       - "traefik.frontend.rule=Path:/api/gsuite-wrapper/healthcheck;PathPrefixStrip:/api/gsuite-wrapper"
+      - "traefik.frontend.rule=Path:/api/gsuite-wrapper/api-docs.json;PathPrefixStrip:/api/gsuite-wrapper"
```

Comment:
won't that break the healthcheck exposing to public? as the traefik.frontend.rule is overridden

Reply:
no, it is a different rule for what i remember (i will re-test with more attention in my setup)

Reply:
good catch